server/src/infrastructure/octopod.js

```diff
@@ -29,20 +29,19 @@ const OctopodClient = {
     });
   },
 
```

Comment:
Commentaire général sur ce fichier : pourquoi est-il appelé octopod.js alors que quand tu l'importes tu nommes ta variable octopodClient ?
=> to rename octopod-client.js